src/remotelog.lua

```diff
@@ -39,15 +42,18 @@ local socket = require("socket")
 --
 -- @param timestamp_pattern layout of timestamps displayed in the logs
 --
--- @param use_high_resolution_time switch high resolution time display on or off
+-- @param use_high_resolution_time switch high resolution time display on or off (default)
+--
+-- @param fallback_strategy what to do if the remote listener connection cannot be established (default: console log)
 --
 -- @return module loader
 --
-function M.init(timestamp_pattern, use_high_resolution_time)
+function M.init(timestamp_pattern, use_high_resolution_time, fallback_strategy)
     if timestamp_pattern then
         M.timestamp_pattern = timestamp_pattern
     end
     M.use_high_resolution_time = use_high_resolution_time or false
+    M.fallback_strategy = fallback_strategy or fallback_strategies.CONSOLE
```

Comment:
Duplicate code. Is it possible to call `set_fallback_strategy()` here?

Reply:
Done.